docs/deep-dive/03-dd-expose-function-serverless.md

```diff
@@ -12,6 +12,8 @@ When you complete this tutorial, you get a Function that:
 
 >**NOTE:** To learn more about securing your Function, see the [tutorial](/components/api-gateway#tutorials-expose-and-secure-a-service-deploy-expose-and-secure-the-sample-resources).
 
+> **TIP:** Read also about [Function’s specification](#details-function-s-specification) if you are interested in its signature, `event` and `context` objects, and custom HTTP responses the Function returns.
+
 ## Prerequisites
 
 This tutorial is based on an existing Function. To create one, follow the [Create a Function](#tutorials-create-a-function) tutorial.
```

Comment:
```suggestion
This tutorial is based on the existing Function. To create one, follow the [Create a Function](#tutorials-create-a-function) tutorial.
```

Reply:
Is it really a prerequisite?

Reply:
I believe so. I'm not sure if "the" is ok here. An existing but any ...